src/govuk/components/notification-banner/notification-banner.yaml

```diff
@@ -1,48 +1,48 @@
-params:
+–params:
```

Comment:
```suggestion
params:
```

Reply:
@vanitabarrett Hi! Before I commit this — what's it for? (Confession: I don't know much about yaml...this edit is for an under-the-hood thing that won't be visible to users, is that right?)

Reply:
Hi @EoinShaughnessy - yep, that's right 👍 In yaml, a dash signifies a list item, so in this file we have multiple list items within a 'params' object. As params is the parent object, I don't think it makes sense for it to have a dash and other yaml files follow that pattern too.

Reply:
@vanitabarrett That makes sense! Ta for explaining!